### PR TITLE
feat(db): connect

### PR DESCRIPTION
**Pull request by bmclear**
- Add basic mongodb connection logic
- Replace mongoose with mongodb

### CODE REVIEW

**Comment by bmclear** [comment]
Not sure how you want to do branches/merging or anything, but I figured I could at least put this code up for you to look at. Really basic stuff, just replace <password> with the actual password when you try to connect. From there, we should be able to set up basic operations.

I wasn't sure what to do with the index.js and index.js.map files; let me know if I should be rebuilding those somehow. And of course, any feedback is appreciated.

File: src/index.ts [hunk 1]
```diff
@@ -1,18 +1,19 @@
 import Koa = require('koa');
-import mongoose = require('mongoose');
 
-const app = new Koa();
+const MongoClient = require('mongodb').MongoClient;
+const uri = "mongodb+srv://test-user_1:<password>@cluster0-dojin.mongodb.net/test?retryWrites=true";
```
Comment (JoshuaKGoldberg) [inline src/index.ts]: > `<password>`

Summarizing conversation: we'll want to look to look at local password storage. Will do soon!

Comment (JoshuaKGoldberg) [reply src/index.ts]: More conversation: we'll want to the atlas db or a local db as per dev or production builds.

**Comment by bmclear** [comment]
🚢 it

**Pull request merged**